src/cart/crt_hg.c

```diff
@@ -885,7 +885,7 @@ crt_hg_req_create(struct crt_hg_context *hg_ctx, struct crt_rpc_priv *rpc_priv)
 		}
 	}
 
-	if (crt_gdata.cg_sep_mode == true) {
+	if (crt_gdata.cg_sep_mode == true && rpc_priv->crp_hg_hdl) {
```

Comment:
@frostedcmos is this one OK with you?

Reply:
If coverity thinks it can be NULL then it'll show why it thinks that which would be easier to trace than adding a assert().  Perhaps leave this out the PR and assign the ticket to someone.